src/util.ts

```diff
@@ -997,6 +1010,21 @@ export function isTokenDetectionSupportedForNetwork(chainId: string): boolean {
   );
 }
 
+/**
+ * Check if token list polling is enabled for a given network.
+ * Currently this method is used to support e2e testing for consumers of this package.
+ *
+ * @param chainId - ChainID of network
+ * @returns Whether the current network supports tokenlists
+ */
+export function isTokenListSupportedForNetwork(chainId: string): boolean {
+  const chainIdDecimal = convertHexToDecimal(chainId).toString();
+  return (
+    isTokenDetectionSupportedForNetwork(chainIdDecimal) ||
+    chainIdDecimal === GANACHE_CHAIN_ID
```

Comment:
isTokenDetectionSupportedForNetwork was preventing this [e2e test on the extension](https://github.com/MetaMask/metamask-extension/blob/develop/test/e2e/tests/add-hide-token.spec.js#L82) from testing the import-token flow that uses token-lists since it returns false for the ganache local network chainId (1337) [preventing the controller from polling](https://github.com/MetaMask/controllers/blob/5b1339736410cbb852b3b1f6acf08f039ecf8312/src/assets/TokenListController.ts#L148) and adding a token-list to state. This test is meant to mimic the behavior of Mainnet but on a local ganache network. We currently mock the network request to our tokenlist proxy for the e2e test [here](https://github.com/MetaMask/metamask-extension/blob/7d1259476bf3edfae38df3bd19f209dbd7a6e6c4/test/e2e/mock-e2e.js#L128)